libs/mocks/README.md

```diff
@@ -9,7 +9,7 @@ Network mocking for testing the WordPress mobile apps
 To use this library in your project, you must set it up as a subtree.
 From the root of your main project, add the subtree:
 
-    $ git subtree add --prefix=libs/mocks git@github.com:wordpress-mobile/WordPressMocks.git develop
+    $ git subtree add --prefix=libs/mocks git@github.com:wordpress-mobile/WordPressMocks.git trunk
```

Comment:
IIRC this is no longer value as we've removed this subtree in favour of [storing the mocks right in the project](https://github.com/wordpress-mobile/WordPress-Android/tree/develop/libs/mocks), so I think we can get rid of this entirely?

Reply:
@jkmassel Would you mind opening a separate PR for that? I don't think this is the right place to make that change.

Reply:
Just to be clear, the reason I am asking you to open the PR instead of opening it myself - which I'd be happy to do - is because you have some context about this issue and I think it's important to include that in the PR description.

I also thought that we were still using this module, for the UI tests, so I'm curious to see if you have any issues there.